modules/sonar/Makefile

```diff
@@ -0,0 +1,40 @@
+## #####################################
+##
+## Module: Sonar
+##
+## This makefile contains common instructions to scan projects with SonarQube
+
+## Variables
+
+## PROJ_VERSION is the version of the project to be shown in Sonar. It defaults 
+## to local, but in production you may want to use the deployment's version
+PROJ_VERSION="${SONAR_PROJ_VERSION:-local}"
+
+## SONAR_AUTH_TOKEN is the token you generate to authenticate with your Sonar 
+## deployment. You can see how to generate and use one here: 
+## https://docs.sonarqube.org/latest/user-guide/user-token/
+## THIS TOKEN SHOULD BE KEPT A SECRET
+
+## SONAR_URL is the URL for your Sonar deployment. While showing this value is 
+## not necessarily dangerous, it is recommended that you keep it a secret
+
+include $(STARK_BUILD_DIR)/modules/golang/Makefile 
```

Comment:
This causes the golang Makefile be included twice. Please remove it. 

- We can assume the go module is already included (like the docker module does); or
- WE can just test if GO_MODULE is not defined and fail (docker module also does that)

Example:
```
ifndef GO_MODULE
	$(error Please set GO_MODULE variable or import go module before importing sonar)
endif

Reply:
that's way better, i'll do that